pymc/backends/arviz.py

```diff
@@ -63,6 +63,22 @@ def find_observations(model: Optional["Model"]) -> Optional[Dict[str, Var]]:
     return observations
 
 
+def find_constants(model: Optional["Model"]) -> Optional[Dict[str, Var]]:
```

Comment:
I think you needs something more strict as in this method: https://github.com/pymc-devs/pymc/blob/162ad6c3d1cc08613ba61e2851e8de253878ebf9/pymc/backends/arviz.py#L468-L503

If possible we should refactor it to be a standalone function, so that we don't duplicate the code (assuming the same code can be used here)